schema.sql

```diff
@@ -68,4 +68,13 @@ CREATE TABLE visits (
     PRIMARY KEY (animal_id, vet_id, date_of_visit),
     FOREIGN KEY (animal_id) REFERENCES animals(id),
     FOREIGN KEY (vet_id) REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
+/* Performance */
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Create index
+CREATE INDEX animals_index ON visits(animal_id);
+CREATE index vets_index ON visits(vet_id);
```

Comment:
Even if the execution times difference is very small in your case, I can clearly see that you had some fun with the explain analyze and I am sure that in the future you will know how to start the investigation for some slow DB queries :smile: